app/controllers/manuals_controller.rb

```diff
@@ -22,6 +22,19 @@ def new
     @manual = Manual.new
   end
 
+  def create
+    @manual = Manual.new(params["manual"])
+    @manual.organisation_content_ids = current_user.organisation_content_id
```

Comment:
When we did this we thought it was a pain to have to remember to set things as an array and wanted the setter to handle it.  On reflection I think you're right, this is odd.  Will go back.